### DISCUSSION TITLE
Upgrading Mint from 20.3 to 21

### DISCUSSION THREAD

**Discussion by Tippon** [abraunegg/onedrive#2445] [Q&A] [answered]
I'm upgrading the distro using mintupgrade, and it's asking me to remove the Onedrive repository. Should I uninstall then reinstall Onedrive, or is there a better way to upgrade along with the distro?

Thanks in advance :)

**Comment by abraunegg** [comment, marked as answer]
@Tippon 
The process you should following is the following:

* Remove the 'onedrive' package
* Remove any configured repository where the 'onedrive' package was installed from
* Upgrade your distribution
* Install the 'onedrive' package following these instructions: https://github.com/abraunegg/onedrive/blob/master/docs/ubuntu-package-install.md

**Reply by Tippon** [reply]
Brilliant, thank you :)
